packages/calling-stateful-client/src/CallDeclarative.ts

```diff
@@ -99,5 +50,5 @@ export const callDeclaratify = (call: Call, context: CallContext): DeclarativeCa
     value: () => proxyCall.unsubscribe()
   });
 
-  return new Proxy(call, proxyCall) as DeclarativeCall;
+  return new Proxy(call, proxyCall) as any as DeclarativeCall;
```

Comment:
similar here, why as any?

Reply:
Hmmm... forget why added this(I guess it was because there was some type problems there before I complete the whole type dependencies), removed